### ISSUE TITLE
accessibility rules - check for missing alt attributes

### ISSUE THREAD

**Issue by ckundo** [causes/haml-lint#69]
Have you considered adding some basic accessibility assertions to your rules (i.e. presence of `alt` attributes, `label` association on form elements)?

I'd like to work on some PRs for accessibility rules, starting with looking for missing `alt` attributes. Would you be open to that?

**Comment by sds** [comment]
Hey @ckundo,

Thanks for suggesting ways to improve `haml-lint`. We we would be open to accepting pull requests for some accessibility-related lints. However, depending on how noisy those lints are we may end up disabling them by default so they are opt-in.

The concern here is that HAML may not be the right place to do this kind of enforcement. There are some uses of HAML which the intended consumer is not a user-facing website, so such lints would not be useful.

I'm curious to see what you are proposing. Ideally, any linters would be backed by a relevant RFC or W3C standard of some sort, and not just be some "best practice."

Depending on the scope of these linters we might also end up namespacing them so they can be enabled/disabled wholesale with something like:

```yaml
linters:
  Accessibility::*:
    enabled: true
```

Interested in seeing what you have to propose.

**Comment by ckundo** [comment]
Hey @sds, I submitted a PR, I welcome your review and feedback. Thanks!

**Comment by sds** [comment]
Implemented by #70. Thanks!

**Comment by ckundo** [comment]
Awesome, thanks @sds!
